youtube_dlc/extractor/youtube.py

```diff
@@ -2051,7 +2051,7 @@ def _extract_filesize(media_url):
 
                 if cipher:
                     if 's' in url_data or self._downloader.params.get('youtube_include_dash_manifest', True):
-                        ASSETS_RE = r'"assets":.+?"js":\s*("[^"]+")'
+                        ASSETS_RE = r'(?:"assets":.+?"js":\s*("[^"]+"))|(?:"jsUrl":\s*("[^"]+"))'
```

Comment:
you can also use **"js(?:Url)?"** to catch both variants: "js" and "jsUrl"

Reply:
Yee this worked for me! thanks!

Reply:
Sure, that would also work. I'm a bit concerned about possible false positives when just looking for "js", but that might be a bit paranoid of me. I don't think it makes a huge difference either way.